### PR TITLE
SetInteriorVD with data not related to defined module

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Script with the check of the SetInteriorVD request with module data which are not related to defined module.

### CODE REVIEW

File: test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/024_params_for_another_module_in_moduleData.lua [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+-- Description:
+-- In case:
+-- 1) RC app sends SetInteriorVehicleData(CLIMATE) request with parameters are related to RADIO module
+-- SDL must: 
+-- 1) Transfer request to HMI and cut the data related to RADIO module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
```
Comment (dboltovskyi) [inline test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/024_params_for_another_module_in_moduleData.lua]: @GetmanetsIrina `commonTestCases` is not used

File: test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/024_params_for_another_module_in_moduleData.lua [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		moduleData = moduleData
+	})
+
+	EXPECT_HMICALL("RC.SetInteriorVehicleData",	{
+		appID = commonRC.getHMIAppId(),
+		moduleData = commonRC.getSettableModuleControlData(moduleType)
+	})
+	:Do(function(_, data)
+			commonRC.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {
+				moduleData = commonRC.getSettableModuleControlData(moduleType)
```
Comment (dboltovskyi) [inline test_scripts/RC/CLIMATE_RADIO/SetInteriorVehicleData/024_params_for_another_module_in_moduleData.lua]: @GetmanetsIrina Please use the same indentation character across the script. E.g. in line `40` tabs are used, however in line `45` - spaces.
I propose to use spaces: tab width is 2 spaces.

**Comment by dboltovskyi** [comment]
The internal review iteration has been moved to the LuxoftSDL.

**Comment by VNikiforov-dev** [comment]
reopened PR with update: #2621

**Pull request closed without merging**